src/emmc.c

```diff
@@ -772,9 +774,14 @@ pu_emmc_parse_partitions(PuEmmc *emmc,
     emmc->partitions = g_list_reverse(emmc->partitions);
 
     if (emmc->num_expanded_parts > 0) {
-        emmc->expanded_part_size = (emmc->device->length - fixed_parts_size
-                                    - 2 * emmc->num_logical_parts)
-                                    / emmc->num_expanded_parts;
+        emmc->expanded_part_size = emmc->device->length - fixed_parts_size
+                                    - 2 * emmc->num_logical_parts;
+
+        /* Reserve blocks for secondary GPT */
+        if (g_str_equal(emmc->disktype->name, "gpt")) {
+            emmc->expanded_part_size -= PARTITION_TABLE_SIZE_GPT;
+        }
+        emmc->expanded_part_size /= emmc->num_expanded_parts;
```

Comment:
Please add one empty line after the `if()` statement, as it should be logically separated from the final division calculating the expanded partition size.